cmd/gapit/flags.go

```diff
@@ -369,9 +369,10 @@ type (
 		CaptureFileFlags
 	}
 	GetTimestampsFlags struct {
-		Gapis GapisFlags
-		Gapir GapirFlags
-		Out   string `help:"output file to save the profiling result"`
+		Gapis     GapisFlags
+		Gapir     GapirFlags
+		LoopCount int    `help:"The number of times to loop the trace."`
```

Comment:
We should probably hide this until it is no longer experimental. If you add a _ to the start of the help message it will be hidden by default.

`help:"_The number of times to loop the trace. (experimental)"`